pkg/ami/ssm_resolver.go

```diff
@@ -83,6 +86,10 @@ func MakeSSMParameterName(version, instanceType, imageFamily string) (string, er
 // MakeManagedSSMParameterName creates an SSM parameter name for a managed nodegroup
 func MakeManagedSSMParameterName(version string, amiType ekstypes.AMITypes) (string, error) {
 	switch amiType {
+	case ekstypes.AMITypesAl2023X8664Standard:
+		return fmt.Sprintf("/aws/service/eks/optimized-ami/%s/%s/x86_64/standard/recommended/release_version", version, utils.ToKebabCase(api.NodeImageFamilyAmazonLinux2023)), nil
+	case ekstypes.AMITypesAl2023Arm64Standard:
```

Comment:
Super nit, maybe in the future we can make these names consistent with something similar to `AMITypesAL2023ARM64Standard`.

ACK has a library for making those kind of transformations https://github.com/aws-controllers-k8s/pkg/blob/main/names/names.go#L44-L191 - maybe one day we'll dedicated a Go module for it